tests/systemtests/query_test.go

```diff
@@ -0,0 +1,81 @@
+//go:build system_test
+
+package systemtests
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"io"
+	"net/http"
+	"strconv"
+	"testing"
+
+	"cosmossdk.io/systemtests"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRestQueries(t *testing.T) {
```

Comment:
IMHO this feature is fully covered by the e2e test. I have committed this system test temporary to let people try with a running server. This test would be removed before merging